### DISCUSSION TITLE
RFM69 and OOK

### DISCUSSION THREAD

**Discussion by cpainchaud** [jgromes/RadioLib#263] [Q&A] [answered]
Hi gents!

 I am looking for example with RF69 and OOK, i am having trouble to make it work and it while the parts are there it seems it's not doable with actual code.

thank you!

**Comment by jgromes** [comment, marked as answer]
Seems like OOK is not supported on RF69 - I don't think there's any particular reason for it, it just isn't implemented.

**Comment by cpainchaud** [comment]
thx for taking the time to answer!
